lib/client.c

```diff
@@ -1072,6 +1088,9 @@ SF_STATUS STDCALL snowflake_set_attribute(
         case SF_CON_AUTHENTICATOR:
             alloc_buffer_and_copy(&sf->authenticator, value);
             break;
+        case SF_CON_OAUTH_TOKEN:
```

Comment:
Please add to snowflake_get_attribute() as well.

Reply:
Done.